tetrateci/tsb-util/tsb_util.py

```diff
@@ -0,0 +1,358 @@
+import urllib.request
+import os, sys, shutil, platform
+import argparse
+import config
+import certs
+from subprocess import PIPE, Popen
+from jinja2 import Template
+import yaml
+
+def save_file(fname, content):
+    f = open(fname, "w")
+    f.write(content)
+    f.close()
+
+def create_namespace(ns, labels, fname):
```

Comment:
to be consistent, take the key as parameter and create the filename inside the func.